keps/sig-node/2837-pod-level-resource-spec/README.md

```diff
@@ -0,0 +1,1108 @@
+Yes, the PodSpec object will be modified to include new fields for pod level resources
+specifications i.e. pod.spec.resources.requests and pod.spec.resources.limit.
+
+
+###### Will enabling / using this feature result in any new calls to the cloud provider?
+
+<!--
+Describe them, providing:
+  - Which API(s):
+  - Estimated increase:
+-->
+No
+
+###### Will enabling / using this feature result in increasing size or count of the existing API objects?
+
+<!--
+Describe them, providing:
+  - API type(s):
+  - Estimated increase in size: (e.g., new annotation of size 32B)
+  - Estimated amount of new objects: (e.g., new Object X for every existing Pod)
+-->
+Negligible.
+
+###### Will enabling / using this feature result in increasing time taken by any operations covered by existing SLIs/SLOs?
+
+<!--
+Look at the [existing SLIs/SLOs].
+
+Think about adding additional work or introducing new steps in between
+(e.g. need to do X to start a container), etc. Please describe the details.
+
+[existing SLIs/SLOs]: https://git.k8s.io/community/sig-scalability/slos/slos.md#kubernetes-slisslos
+-->
+
+The computational overhead introduced in API server, scheduler and kubelet due to additional validation for pod level resource specifications should be negligible. Thorough testing and monitoring will ensure the SLIs/SLOs are not impacted.
+
+###### Will enabling / using this feature result in non-negligible increase of resource usage (CPU, RAM, disk, IO, ...) in any components?
+
+<!--
+Things to keep in mind include: additional in-memory state, additional
+non-trivial computations, excessive access to disks (including increased log
+volume), significant amount of data sent and/or received over network, etc.
+This through this both in small and large cases, again with respect to the
+[supported limits].
+
+[supported limits]: https://git.k8s.io/community//sig-scalability/configs-and-limits/thresholds.md
+-->
+
+No. Pod-level resource information could lead to marginal increase in storage requirements for the new fieleds, although
+its impact is likely to be small compared to other Kubernetes object data.
+
+###### Can enabling / using this feature result in resource exhaustion of some node resources (PIDs, sockets, inodes, etc.)?
+
+<!--
+Focus not just on happy cases, but primarily on more pathological cases
+(e.g. probes taking a minute instead of milliseconds, failed pods consuming resources, etc.).
+If any of the resources can be exhausted, how this is mitigated with the existing limits
+(e.g. pods per node) or new limits added by this KEP?
+
+Are there any tests that were run/should be run to understand performance characteristics better
+and validate the declared limits?
+-->
+Yes. With pod-level resource limits, it's possible to pack more pods onto a node. This increased
+pod density can also lead to higher consumption of resources like PIDs and sockets, as each
+pod and its containers create processes and potentially establish network resources.
+
```

Comment:
I would also say that the maxPods field should already account for and limit the problems of this

Reply:
was it about mode pods or more containers? I wonder if we will see a growing number of Pods with many containers. Since you don't have to specify requests and limits on individual containers, people may start packing more low traffic containers into Pods. But I think it is more suitable for the risks section